### ISSUE TITLE
npm release 1.1.0 does not pull in babel-polyfill

### ISSUE THREAD

**Issue by m-schmoock** [PlayNetwork/json-ipc-lib#9]
@brozeph the new release 1.1.0 does not correctl pull in `babel-polyfill` as dependency package when installed via npm in another project.

```
Error: Cannot find module 'babel-polyfill'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:609:15)
    at Function.Module._load (internal/modules/cjs/loader.js:535:25)
    at Module.require (internal/modules/cjs/loader.js:663:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/home/foo/bar/node_modules/json-ipc-lib/dist/index.js:7:1)
```

**Comment by m-schmoock** [comment]
when injected manually we get missing `source-map-support/register`

**Comment by brozeph** [comment]
Odd, should be pulling `@babel/polyfill`... will look!

**Comment by brozeph** [comment]
So after some research, there’s no need for pulling in polyfill at all. I need to rework a couple things, but I’m pushing towards using @babel/runtime instead. Polyfill is presently in for test runs, so I want to adjust that too. Also, I don’t think I have the runtime pieces set up correctly just yet: <https://babeljs.io/docs/en/babel-plugin-transform-runtime>

**Comment by brozeph** [comment]
Patch is now released as `v1.1.1`
